### ISSUE TITLE
[BUG] Library receives 'Forbidden' when attempting API auth

### ISSUE THREAD

**Issue by afk11** [Pyhive/Pyhiveapi#8]
**Describe the bug**
I recently noticed I no longer saw my Hive thermometer status when I logged into HomeAssistant.
I decided to update HA to the latest version, and re-add my weather patches to see if it fixed it - NO
I then went to my dev clone of this library, and put some debugging print statements during the auth API call

On this part: 
```python
            print(api_resp_p) // added by me
            if ('token' in api_resp_p and
                    'user' in api_resp_p and
                    'platform' in api_resp_p):
```
it turns out that the response from Hive is `{'message': 'Forbidden'}`

**Expected behavior**
There should be a notice in the Home Assistant logs that the pyhiveapi received the Forbidden message on login

It should also log in successfully (my creds are ok)

**Screenshots**
When I try the example, which worked in the past, with my credentials, I get this (some of the output is from my debugging FYI)

```
Attempt to log in to the Hive API
do call
API call finished
{'message': 'Forbidden'}
no token/user/platform in auth



Attempt to get Outside temperature
Outside Temperature = 0.0

Update Data
do call
API call finished
{'message': 'Forbidden'}
no token/user/platform in auth
Update Data result = False
An update result of False = data is cached and a Hive API call was not made
An update result of True = Hive API called data was retrieved and cached

```

**Comment by afk11** [comment]
Oh, FYI, I was testing against a 0.20.2 branch. Based on the PR I submitted for the weather stuff before. afk11/Pyhiveapi branch gracefully-degrade-weather-status

Maybe I should try the 0.3 version, but I don't see an example there anymore. This was what I was using to test the integration without messing with HomeAssistant. 

I also wonder - since you added Azure pipelines maybe you're able to setup encrypted envvars with your hive credentials so an integration test would verify that the plugin works?

**Comment by afk11** [comment]
Oh, I just learned about an issue about this on HA. I guess they're related https://github.com/home-assistant/core/issues/43890

**Comment by KJonline** [comment]
This is fixed in 0.3 but has not yet made it to the home assistant yet. There is a custom component that can be used.

**Comment by afk11** [comment]
Right - I did try the `_b25` beta release, but pretty much got the same result, so I rolled back out of betas for now.

**Comment by KJonline** [comment]
So when you installed via HACs what did you do next. Did you set hive up via the UI?

**Comment by afk11** [comment]
I just put the credentials in the configuration file and imported it from HACS. I don't remember if I did anything extra in the first pass of setting this up. but after i updated HA, I tried switching repos + simply resinstalling and restarting HA

**Comment by KJonline** [comment]
Beta 25 should work it just requires configuring via the UI

**Comment by afk11** [comment]
Ah perfect! I followed this https://github.com/Pyhive/HA-Hive-Custom-Component/#ui Since I saw it picked up devices + entities on the integrations page, I clicked in and saw it called 'climate.zone_1' instead of 'climate.heating' that the Dashboard card used before! So I tweaked that as well, and it's working :+1:  Thanks for your all your work on this! Having to deal with the hive changes must be frustrating.
